tests/unit/lxd/test_remotes.py

```diff
@@ -163,24 +185,34 @@ def test_get_image_remote_xenial_error():
     The remote image for Xenial has not been chosen for craft-providers + LXD, so an
     error is raised.
     """
+    base = ubuntu.BuilddBase(alias=ubuntu.BuilddBaseAlias.XENIAL)
     with pytest.raises(lxd.LXDError) as raised:
-        lxd.remotes.get_remote_image(ubuntu.BuilddBaseAlias.XENIAL.value)
+        lxd.remotes.get_remote_image(base)
 
-    assert str(raised.value) == (
-        "could not find a lxd remote image for the provider base '16.04'"
+    assert "could not find a lxd remote image for the provider base " in str(
+        raised.value
     )
 
 
 def test_get_image_remote_error():
     """Raise an error for an unknown provider base."""
+    base = ubuntu.BuilddBase(alias=-1)  # type: ignore
     with pytest.raises(lxd.LXDError) as raised:
-        lxd.remotes.get_remote_image("unknown-base")
+        lxd.remotes.get_remote_image(base)
 
-    assert str(raised.value) == (
-        "could not find a lxd remote image for the provider base 'unknown-base'"
+    assert "could not find a lxd remote image for the provider base " in str(
+        raised.value
     )
 
 
+def test_get_image_remote_before_2_0_error():
```

Comment:
pedantic: I would use a term like `backwards_compatible` or `deprecated` instead of `before_2_0`